_pages/dev/features/scroll-to-top.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Scroll to Top
+feature:
+  - name: Scroll to Top
+    spa_version: 5.0
+    cx_version: 2105
+---
+
+{% capture version_note %}
+{{ site.version_note_part1 }} 5.0 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+The scroll to top feature allows users to quickly scroll back to the top of the page.
+
+---
+
+**Table of Contents**
+
+- This will become a table of contents (this text will be scrapped).
+{:toc}
+
+---
+
+## Enabling Scroll to Top
+
+Scroll to top is part of the `@spartacus/storefront` library, and installed during the initial setup. To enable this feature in your storefront add the scroll to top CMS data to a content slot using ImpEx. 
+
+### CMS Component
+
+Scroll to top is CMS-driven and consists of the following CMS component:
+
+- `ScrollToTopComponent`
+
+If you are using the version 5.0 or greater of [{% assign linkedpage = site.pages | where: "name", "spartacussampledata-extension.md" %}{{ linkedpage[0].title }}]({{ site.baseurl }}{% link _pages/install/spartacussampledata-extension.md %}), the scroll to top component is already enabled. However, if you decide not to use the `spartacussampledata` extension, you can enable scroll to top through ImpEx.
+
+**Note:** The `$contentCV` variable that is used throughout the following ImpEx examples, and which stores information about the content catalog, is defined as follows:
+
+```text
+$contentCatalog=powertools-spaContentCatalog
+$contentCV=catalogVersion(CatalogVersion.catalog(Catalog.id[default=$contentCatalog]),CatalogVersion.version[default=Staged])[default=$contentCatalog:Staged]
+```
+
+#### Adding Scroll to Top CMS Component Manually
+
+This section describes how to add the scroll to top CMS component to Spartacus using ImpEx.
+
+You can enable the **Scroll to Top** component by adding the CMS data with the following ImpEx:
```

Comment:
Suggestion:

You can create the **Scroll to Top** component with the following ImpEx: